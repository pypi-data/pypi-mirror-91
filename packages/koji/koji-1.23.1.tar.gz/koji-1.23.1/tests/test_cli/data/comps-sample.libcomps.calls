[
  [ "groupListAdd", [ "tag", "additional-devel" ], { "force": null, "description": "Additional development headers and libraries for developing applications", "uservisible": false, "is_default": false, "biarchonly": true, "display_name": "Additional Development", "langonly": "fr" } ],
  [ "groupPackageListAdd", [ "tag", "additional-devel", "alsa-lib-devel" ], { "type": "default", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "additional-devel", "audit-libs-devel" ], { "type": "default", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "additional-devel", "binutils-devel" ], { "type": "default", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "additional-devel", "boost-devel" ], { "type": "default", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "additional-devel", "bzip2-devel" ], { "type": "default", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "additional-devel", "cyrus-sasl-devel" ], { "type": "default", "force": null, "basearchonly": false } ],
  [ "groupListAdd", [ "tag", "backup-client" ], { "force": null, "description": "Client tools for connecting to a backup server and doing backups.", "uservisible": true, "is_default": true, "biarchonly": false, "display_name": "Backup Client", "langonly": null } ],
  [ "groupPackageListAdd", [ "tag", "backup-client", "amanda-client" ], { "type": "mandatory", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "backup-client", "bacula-client" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupListAdd", [ "tag", "backup-server" ], { "force": null, "description": "Software to centralize your infrastructure's backups.", "uservisible": true, "is_default": false, "biarchonly": false, "display_name": "Backup Server", "langonly": null } ],
  [ "groupPackageListAdd", [ "tag", "backup-server", "amanda-server" ], { "type": "mandatory", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "backup-server", "mt-st" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "backup-server", "mtx" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupListAdd", [ "tag", "ansible-node" ], { "force": null, "description": null, "uservisible": true, "is_default": false, "biarchonly": false, "display_name": "Ansible node", "langonly": null } ],
  [ "groupPackageListAdd", [ "tag", "ansible-node", "python2-dnf" ], { "type": "mandatory", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "ansible-node", "libselinux-python" ], { "force": null, "requires": "selinux-policy", "basearchonly": false, "type": "conditional" } ],
  [ "groupListAdd", [ "tag", "d-development" ], { "force": null, "description": "These include development tools and libraries such as ldc, and geany-tag.", "uservisible": true, "is_default": false, "biarchonly": false, "display_name": "D Development Tools and Libraries", "langonly": null } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "ldc" ], { "type": "mandatory", "force": null, "basearchonly": true } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "ldc-druntime" ], { "type": "mandatory", "force": null, "basearchonly": true } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "ldc-druntime-devel" ], { "type": "mandatory", "force": null, "basearchonly": true } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "ldc-phobos-devel" ], { "type": "mandatory", "force": null, "basearchonly": true } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "make" ], { "type": "mandatory", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "pkgconfig" ], { "type": "mandatory", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "ctags" ], { "type": "default", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "indent" ], { "type": "default", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "astyle" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "cmake" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "derelict-devel" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "geany" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "gl3n-devel" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "insight" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "nemiver" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "d-development", "uncrustify" ], { "type": "optional", "force": null, "basearchonly": false } ],
  [ "groupListAdd", [ "tag", "empty-group-1" ], { "force": null, "description": "empty group 1 desc", "uservisible": true, "is_default": false, "biarchonly": false, "display_name": "empty group 1", "langonly": null } ],
  [ "groupListAdd", [ "tag", "empty-group-2" ], { "force": null, "description": "empty group 2 desc", "uservisible": true, "is_default": false, "biarchonly": false, "display_name": "empty group 2", "langonly": null } ],
  [ "groupListAdd", [ "tag", "unknown-group" ], { "force": null, "description": "unknown group desc", "uservisible": true, "is_default": false, "biarchonly": false, "display_name": "unknown group", "langonly": null } ],
  [ "groupPackageListAdd", [ "tag", "unknown-group", "unknown" ], { "type": "unknown", "force": null, "basearchonly": false } ],
  [ "groupPackageListAdd", [ "tag", "unknown-group", "unknown2" ], { "type": "unknown", "force": null, "basearchonly": false } ]
]
