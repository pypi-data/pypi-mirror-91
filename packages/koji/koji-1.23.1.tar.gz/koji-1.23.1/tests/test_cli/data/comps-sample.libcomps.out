Group: additional-devel (Additional Development)
  Package: alsa-lib-devel: {'basearchonly': False, 'type': 'default'}
  Package: audit-libs-devel: {'basearchonly': False, 'type': 'default'}
  Package: binutils-devel: {'basearchonly': False, 'type': 'default'}
  Package: boost-devel: {'basearchonly': False, 'type': 'default'}
  Package: bzip2-devel: {'basearchonly': False, 'type': 'default'}
  Package: cyrus-sasl-devel: {'basearchonly': False, 'type': 'default'}
Group: backup-client (Backup Client)
  Package: amanda-client: {'basearchonly': False, 'type': 'mandatory'}
  Package: bacula-client: {'basearchonly': False, 'type': 'optional'}
Group: backup-server (Backup Server)
  Package: amanda-server: {'basearchonly': False, 'type': 'mandatory'}
  Package: mt-st: {'basearchonly': False, 'type': 'optional'}
  Package: mtx: {'basearchonly': False, 'type': 'optional'}
Group: ansible-node (Ansible node)
  Package: python2-dnf: {'basearchonly': False, 'type': 'mandatory'}
  Package: libselinux-python: {'basearchonly': False, 'requires': 'selinux-policy', 'type': 'conditional'}
Group: d-development (D Development Tools and Libraries)
  Package: ldc: {'basearchonly': True, 'type': 'mandatory'}
  Package: ldc-druntime: {'basearchonly': True, 'type': 'mandatory'}
  Package: ldc-druntime-devel: {'basearchonly': True, 'type': 'mandatory'}
  Package: ldc-phobos-devel: {'basearchonly': True, 'type': 'mandatory'}
  Package: make: {'basearchonly': False, 'type': 'mandatory'}
  Package: pkgconfig: {'basearchonly': False, 'type': 'mandatory'}
  Package: ctags: {'basearchonly': False, 'type': 'default'}
  Package: indent: {'basearchonly': False, 'type': 'default'}
  Package: astyle: {'basearchonly': False, 'type': 'optional'}
  Package: cmake: {'basearchonly': False, 'type': 'optional'}
  Package: derelict-devel: {'basearchonly': False, 'type': 'optional'}
  Package: geany: {'basearchonly': False, 'type': 'optional'}
  Package: gl3n-devel: {'basearchonly': False, 'type': 'optional'}
  Package: insight: {'basearchonly': False, 'type': 'optional'}
  Package: nemiver: {'basearchonly': False, 'type': 'optional'}
  Package: uncrustify: {'basearchonly': False, 'type': 'optional'}
Group: empty-group-1 (empty group 1)
Group: empty-group-2 (empty group 2)
Group: unknown-group (unknown group)
  Package: unknown: {'basearchonly': False, 'type': 'unknown'}
  Package: unknown2: {'basearchonly': False, 'type': 'unknown'}
